locales/mr/validation.php

```diff
@@ -13,7 +13,7 @@
 
 return [
     'accepted'             => ':attribute ला स्वीकार केला गेला पाहिजे.',
-    'accepted_if'          => 'The :attribute must be accepted when :other is :value.',
+    'accepted_if'          => ':other हे :value असेल तेव्हा हे फील्ड स्वीकारणे आवश्यक आहे.',
```

Comment:
Shouldn't the `:attribute` placeholder appear?

Reply:
fixed, please check.